### ISSUE TITLE
Assign MIDI message to save presets.

### ISSUE THREAD

**Issue by bonninr** [GrandOrgue/grandorgue#2074]
Greetings,

When creating headless VPOs, a missing feature is the ability to save the current presets persistently, probably assigning one MIDI message to the file/save method. 

There are other parts of the GUI which I think could also benefit from having a dedicated MIDI message, like the release tail up/down and polyphony up/down.

**Comment by larspalo** [comment]
> ability to save the current presets persistently

This already exist. The save button is on the "Master Controls" panel that's visible when you have an organ loaded/opened.

> There are other parts of the GUI which I think could also benefit from having a dedicated MIDI message, like the release tail up/down and polyphony up/down.

Personally, I just set them to the desired value and then never fiddle with them ever again as there's no need to. The release tail shortening is saved with the preset.

**Comment by bonninr** [comment]
Thanks Lars,

Regarding release tail shortening, I'd like to select one option up or down, and probably send a sysex with the current value.

**Comment by bonninr** [comment]
Just to clarify, does the Save Settings command save all current stop combinations, temperament adjustments, etc?

**Comment by larspalo** [comment]
> Just to clarify, does the Save Settings command save all current stop combinations, temperament adjustments, etc?

Yes, it's the same as the Save option (Ctrl+S) in the File menu and it will only act on the selected preset (number). If you just want combinations saved though, you'll need to export them through the menu or via the Combination setter panel.

> Regarding release tail shortening, I'd like to select one option up or down, and probably send a sysex with the current value.

Just to be clear, "release tail shortening" is actually not at all the same as controlling the amount of reverb (because no reverb is added - the existing release is just truncated and faded out).

In my opinion, a way to control the IR reverb available in the File->Settings Reverb tab in real time (including by MIDI messages) would be much more worthwhile. Setting the release tail shortening is kind of a one time operation whereas adapting the amount of added reverb is something at least I do much more often depending on what I want to hear... Others might of course use the feature differently.

Another thing to consider is that the release tail shortening isn't just a global setting, it can be configured for any level (organ, windchest, rank, pipe) and is effectively used to adapt samples recorded in different rooms (coming from different sample sets) to function in a composite sample set.

**Comment by bonninr** [comment]
Great, thanks, Lars. I was just working on a GUI automation script that receives MIDI signals to trigger the file/save option; now, it won't be necessary.
